ihmc-perception/src/main/java/us/ihmc/sensors/ZEDImageSensor.java

```diff
@@ -257,7 +265,7 @@ private RawImage slMatToRawImage(Pointer slMatPointer, PixelFormat imagePixelFor
                                       sl_mat_get_ptr(slMatPointer, SL_MEM_GPU),
                                       sl_mat_get_step_bytes(slMatPointer, SL_MEM_GPU));
       return new RawImage(null,
-                          imageGpuMat,
+                          imageGpuMat.clone(),
```

Comment:
What does cloning do?

Reply:
It performs a deep copy of the GpuMat and returns the copy